service.subtitles.napiprojekt/service.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*- 
+
+import os
+import re
+import shutil
+import sys
+import urllib
+from urllib.parse import unquote
+
+import unicodedata
+import xbmc
+import xbmcaddon
+import xbmcgui
+import xbmcplugin
+import xbmcvfs
+import hashlib
+
+try:
+    # Python 2.6 +
+    from hashlib import md5
+except ImportError:
+    # Python 2.5 and earlier
+    from md5 import new as md5
+
+__addon__ = xbmcaddon.Addon()
+__scriptid__ = __addon__.getAddonInfo('id')
+__scriptname__ = __addon__.getAddonInfo('name')
+__version__ = __addon__.getAddonInfo('version')
+__language__ = __addon__.getLocalizedString
+
+__cwd__ = xbmc.translatePath(__addon__.getAddonInfo('path'))
+__profile__ = xbmc.translatePath(__addon__.getAddonInfo('profile'))
+__resource__ = xbmc.translatePath(os.path.join(__cwd__, 'resources', 'lib'))
+__temp__ = xbmc.translatePath(os.path.join(__profile__, 'temp', ''))
+
+sys.path.append(__resource__)
```

Comment:
sorry, you can't use sys.path.append. Please use relative module imports:

`from resources.lib import yourmodule`